src/app/main/component/ubs/ubs.module.ts

```diff
@@ -46,6 +47,7 @@ import { SharedModule } from 'src/app/shared/shared.module';
     ReactiveFormsModule,
     IMaskModule,
     MatGoogleMapsAutocompleteModule,
```

Comment:
If we don't use MatGoogleMapsAutocompleteModule, maybe remove?

Reply:
Removed